list-navigation/lambda/handlers/base-api-handler.ts

```diff
@@ -21,5 +21,5 @@ export abstract class BaseApiHandler implements RequestHandler {
         return util.isApiRequest(handlerInput, this.apiName);
     }
 
-    abstract handle(handlerInput : HandlerInput): Response;
```

Comment:
don't think the handle method can actually return anything, can we specifying the real type? I think `Promise<Response>`?

Reply:
Both the record-event handlers, the EndSessionHandler as well as the ConvertOrdinalToIndexHandler extends the BaseApiHandler class. So specifying the type as `Promise<Response>` would allow the return type to be `Response` for them. But I can always make all the handlers async for uniformity.